aws_vpc_policies/aws_security_group_unused_security_group.yml

```diff
@@ -0,0 +1,141 @@
+AnalysisType: policy
+Filename: aws_security_group_unused_security_group.py
+PolicyID: AWS.SecurityGroup.UnusedSecurityGroup
+Enabled: true
+ResourceTypes:
+  - AWS.EC2.SecurityGroup
+Tags:
+  - AWS
+  - PCI
+Reports:
+  PCI:
+    - 1.1.4
+    - 1.2.1
+    - 8.2.1
```

Comment:
Just double checking, are these PCI mappings correct? Just want to make sure they didn't end up here vvia copy/paste.

Reply:
Removed the PCI